src/logistics/definitions.ts

```diff
@@ -57,60 +58,435 @@ export type CargoDefinition = BaseCargoDefinition | UnitCargoDefinition
  */
 export const allCargoDefinitions = createCargoDefinitionsArray(
   {
-    displayName: 'Base Creation Kit',
+    displayName: 'Base Creation Kit', // arg[0]
     id: 'internal-base-create',
     mass: 2000, // TODO: find a reasonable value here
     internal: true,
     type: CargoType.BaseCreate,
     typeName: CargoTypeName.UH1HCargo,
   },
   {
-    displayName: 'Base Upgrade Kit (small)',
+    displayName: 'Base Upgrade Kit (small)', // arg[1]
     id: 'internal-base-upgrade-small',
     mass: 100, // TODO: find a reasonable value here
     internal: true,
     type: CargoType.BaseUpgrade,
     typeName: CargoTypeName.UH1HCargo,
   },
   {
-    displayName: 'Base Upgrade Kit (large)',
+    displayName: 'Base Upgrade Kit (large)', // arg[2]
     id: 'internal-base-upgrade-large',
     mass: 100, // TODO: find a reasonable value here
     internal: false,
     type: CargoType.BaseUpgrade,
     typeName: CargoTypeName.UH1HCargo,
   },
   {
-    displayName: 'MBT M1A2 Abrams',
-    id: 'internal-unit-create-m1a2-abrams',
-    mass: 100, // TODO: find a reasonable value here
+    displayName: 'Truck M939 Heavy', // 2 crates E arg[3]
+    id: 'internal-unit-create-m939-truck',
+    mass: 1000, 
     internal: true,
     type: CargoType.UnitCreate,
     typeName: CargoTypeName.UH1HCargo,
-    unitTypeName: UnitTypeName.M1A2,
+    unitTypeName: UnitTypeName.M939,
+  },
+  {
+    displayName: 'EWR AN/FPS-117 Radar', // 3 crates E arg[4]
+    id: 'internal-unit-create-fps117-ewr',
+    mass: 1000, 
+    internal: true,
+    type: CargoType.UnitCreate,
+    typeName: CargoTypeName.UH1HCargo,
+    unitTypeName: UnitTypeName.FPS117,
+  },
+  {
+    displayName: 'JTAC HMMWV', // 1 crate I/E arg[5]
```

Comment:
>1 crate I/E

this seems like a personal note, it's not clear how it relates to this codebase. We should remove this extra "noise" in the comments here

> arg[5]

this bit can stay, it seems useful

Reply:
et. all this file

Reply:
I add those comments for future use of internal/external crates and keep it in mind.
I will remove them.